contracts/bridge/BridgeTransferCommon.sol

```diff
@@ -12,10 +12,10 @@ contract BridgeTransfer is BridgeHandledRequests, BridgeFee, BridgeOperator {
     bool public isRunning;
 
     uint64 public requestNonce;
-    uint64 public lastHandledRequestBlockNumber;
+    uint64 public sequentialHandledRequestBlockNumber;
     uint64 public sequentialHandleNonce;
     uint64 public maxHandledRequestedNonce;
-    mapping(uint64 => bool) public handledNonces;  // <handled nonce> history
+    mapping(uint64 => uint64) public handledHistory;  // <request nonce> => <request blockNum>
```

Comment:
Why don't you keep `handledNonces` and make `handledRequestBlockNum`?
In this case, you can remove old history of `handledRequestBlockNum`.

Reply:
As we check gas fees of the uint8 and uint64 are the same.

Reply:
handledNoncesToBlockNums

Reply:
Fixed. PTAL.